tests/UKHO.S100PermitService.API.FunctionalTests/Helpers/TestConfiguration.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Microsoft.Extensions.Configuration;
+
+namespace UKHO.S100PermitService.API.FunctionalTests.Helpers
```

Comment:
Can we change the namespace to UKHO.S100PermitService.API.FunctionalTests.Configuration please? Avoid using 'helpers' to name things, as it doesn't conform to SOLID principles well.

Reply:
Have renamed the namespace, will follow this practice ahead as well. Thanks!